fenix/app/src/androidTest/java/org/mozilla/fenix/ui/robots/BrowserRobot.kt

```diff
@@ -590,16 +591,22 @@ class BrowserRobot {
         }
     }
 
-    fun verifyCookiesProtectionHintIsDisplayed(isDisplayed: Boolean) {
-        assertItemContainingTextExists(
-            totalCookieProtectionHintMessage,
-            totalCookieProtectionHintLearnMoreLink,
-            exists = isDisplayed,
-        )
-        assertItemWithDescriptionExists(
-            totalCookieProtectionHintCloseButton,
-            exists = isDisplayed,
-        )
+    fun verifyCookiesProtectionHintIsDisplayed(composeTestRule: HomeActivityComposeTestRule, isDisplayed: Boolean) {
+        if (isDisplayed) {
+            composeTestRule.onNodeWithTag("tcp_cfr.message").assertIsDisplayed()
+            composeTestRule.onNodeWithTag("tcp_cfr.action").assertIsDisplayed()
+        } else {
+            composeTestRule.onNodeWithTag("tcp_cfr.message").assertDoesNotExist()
+            composeTestRule.onNodeWithTag("tcp_cfr.action").assertDoesNotExist()
+        }
+    }
+
+    fun clickTCPCFRLearnMore(composeTestRule: HomeActivityComposeTestRule) {
+        composeTestRule.onNodeWithTag("tcp_cfr.action").performClick()
+    }
+
+    fun dismissCFRPopup(composeTestRule: HomeActivityComposeTestRule) {
```

Comment:
Is this specific to all CFRs ?
If it's only for the TCP CFR, I would rename it.